.github/workflows/integration.yaml

```diff
@@ -107,7 +107,7 @@ jobs:
           TEST_VERSION_UPGRADE_MIN_RELEASE: "1.31"
           TEST_MIRROR_LIST: '[{"name": "ghcr.io", "port": 5000, "remote": "https://ghcr.io", "username": "${{ github.actor }}", "password": "${{ secrets.GITHUB_TOKEN }}"}, {"name": "docker.io", "port": 5001, "remote": "https://registry-1.docker.io", "username": "", "password": ""}, {"name": "rocks.canonical.com", "port": 5002, "remote": "https://rocks.canonical.com/cdk"}]'
         run: |
-          cd tests/integration && sg lxd -c 'tox -e integration -- --tags pull_request'
+          cd tests/integration && sg lxd -c 'tox -e integration --'
```

Comment:
Run the full test suite to ensure stuff does not break. (We have a card to automatically do this for release branches in the future).

Reply:
Remove once the tests pass.

Reply:
resolved